applications/FluidDynamicsApplication/custom_elements/stokes_3D_twofluid.h

```diff
@@ -214,9 +214,10 @@ class Stokes3DTwoFluid
     {
         KRATOS_TRY
 
-        KRATOS_THROW_ERROR(std::logic_error,"method not implemented yet","");
+        MatrixType temp = Matrix();
+        CalculateLocalSystem(temp, rRightHandSideVector, rCurrentProcessInfo);
```

Comment:
I think this is quite inefficient

an better idea @rubenzorrilla ?

Reply:
Hmm, I thought that you would need the LHS to compute RHS, but as this is a symbolic element, you actually don't... Anyway, I am not sure if it's really worthy to refactor the element for this...

Reply:
It is very inefficient.

It should be easy to properly implement it. At the end it's just copying and pasting what is done in the `CalculateLocalSystem` for the RHS in a new method.

Reply:
Wait, I think in this case it's not possible because of the local condensation. I'd need to have a detailed look onto it.

Reply:
I think its possible. Enrichment is independent from LHS so you only need contribution from here.
https://github.com/KratosMultiphysics/Kratos/blob/8bc2be6f6eda4f11d65f1babdbf4b3177bf6ea2e/applications/FluidDynamicsApplication/custom_elements/stokes_3D_twofluid.h#L791
@uksajja If you move what's inside CalculateLocalSystem to a new function you might call this  new function with a calculate_lhs flag so only terms related to rhs are computed. It might be worth a try if it does not overcomplicate the implementation too much...

Reply:
I am not sure if one `if` justifies copy-pasting 
I am guessing the computations in `CalculateRHS` are much more expensive